tests/lean/PPRoundtrip.lean.expected.out

```diff
@@ -55,10 +55,10 @@ decide (1 < 2) ||
 id (fun a => a) 0
 typeAs Nat
   (let x := 1;
-  do 
-    discard (pure 2)
-    let y : Nat := 3
-    pure (x + y))
+  do
+  discard (pure 2)
+  let y : Nat := 3
+  pure (x + y))
```

Comment:
Should we care about this one? Arguably the indent of `do` was wrong to begin with.